lib/storage/src/content_manager/toc.rs

```diff
@@ -380,7 +380,7 @@ impl TableOfContent {
                     ),
                 },
                 // ??? - forwards 3 possible errors: timeout, sender dropped, operation failed
```

Comment:
comment is not outdated

Reply:
fixed in https://github.com/qdrant/qdrant/pull/446/commits/dd40fb2383d8c779066ed3ec52ae18fbc1fbfb79